src/main/java/macrobase/analysis/result/OutlierClassificationResult.java

```diff
@@ -0,0 +1,11 @@
+package macrobase.analysis.result;
+
+import macrobase.datamodel.Datum;
+
+import java.util.List;
+
+public abstract class OutlierClassificationResult {
+
+    public abstract List<Datum> getOutliers();
```

Comment:
This seems appropriate for batch but not tuple-at-a-time processing?

Why not just:
 - Datum
 - isOutlier

Where different methods can optionally pass along a "score": a threshold-based method could have the score, a KDE could have a percentile, etc.